### ISSUE TITLE
myplanet used time calculation giving wrong time

### ISSUE THREAD

**Issue by lmmrssa** [open-learning-exchange/planet#6406]
![image](https://user-images.githubusercontent.com/2909490/82999554-e20f2200-a028-11ea-8d68-bfc6259e8a01.png)

**Comment by paulbert** [comment]
@lmmrssa Yes I asked Leonard what the fields are but forgot to bring it up in the meeting yesterday.

I assumed `totalUsed` was the seconds or milliseconds of time for that session, but that is clearly incorrect so if you or @leomaxi could explain better in this issue that would be very helpful.

**Comment by lmmrssa** [comment]
`totalUsed` is tracking milliseconds of how long device was on in that specific period (day). 
`totalForegroundTime` is how long myplanet was used
We saw there was some issue coming from myplanet when calculating data as phone was not returning `lastTimeUsed` causing time to be negative.
We have raised issue on myPlanet so they put **zero** if there is  negative value. On planet side we need to either cleanup old data with negative `totalUsed` value or we need to ignore them when doing sum.

**Comment by paulbert** [comment]
@lmmrssa The values for `totalUsed` and `totalForegroundTime` is still very strange.  Here's an example on Planet Dev where the difference in the two objects for `totalUsed` is **460578ms** and `totalForegroundTime` is **283111ms** but the difference in `time` is **7ms**.  If the record is created on the same day could there be some overlap?

![image](https://user-images.githubusercontent.com/9203229/83074461-3671cc00-a040-11ea-9988-1fd0f4e3a139.png)
